mmcv/runner/optimizer/default_constructor.py

```diff
@@ -135,7 +139,11 @@ def _is_in(self, param_group, param_group_list):
 
         return not param.isdisjoint(param_set)
 
-    def add_params(self, params, module, prefix='', is_dcn_module=None):
+    def add_params(self,
+                   params: List[dict],
+                   module: torch.nn.Module,
+                   prefix: str = '',
+                   is_dcn_module: Union[int, float, None] = None):
```

Comment:
```suggestion
                   is_dcn_module: Optional[Union[int, float]] = None):
```

Reply:
For multiple types, use `Union[..., None]` instead of `Optional[Union[...]]`
Refers to [Google Style Book](https://github.com/google/styleguide/blob/gh-pages/pyguide.md)
![图片](https://user-images.githubusercontent.com/26739999/170856797-3bd8fa87-7e3c-4d6c-bb52-f99a4d3a7994.png)